brainstormforce/astra
# AST-3779 - Fix:Deprecated warning on Category Archive page with PHP 8.1 or higher.

There is a deprecated warning on the category archive page with PHP 8.1 on the Astra theme. Please check this screenshot: [[Link to the screenshot](https://1drv.ms/i/s!AqrjtHzdcl0_klu_TUPeiBrqo3IX)].

**How to reproduce:**

1. Enable wp_debug in wp-config.php
2. Switch to PHP 8.1 or PHP 8.2
3. Check the category page, and you can see the warning as shown in the above screenshot.
4. Switch back to PHP 8.0, and there is no warning

This does not happen with default themes.

### Types of changes





- Bug Fix - Non breaking changes.

### How has this been tested?


-Test cases mention in the [task](https://brainstormforce.atlassian.net/browse/AST-3779). 

### Checklist:
